packages/backend-modules/republik-crowdfundings/lib/Mail.js

```diff
@@ -1152,4 +1152,79 @@ mail.getPledgeMergeVars = async (
   ]
 }
 
+mail.sendFutureCampaignSenderRewardMail = async (
+  {
+    senderUserId,
+    pledgeUserId,
+    count,
+    hasSenderConsentedToDonate,
+    membershipPeriodEndDate,
+  },
+  { pgdb, t },
+) => {
+  const sender = await pgdb.public.users.findOne({ id: senderUserId })
+  const safeSender = transformUser(sender)
+
+  const pledger = await pgdb.public.users.findOne({ id: pledgeUserId })
+  const safePledger = transformUser(pledger)
+
+  const countNumber = parseInt(count, 10)
```

Comment:
10 is a fallback? what is displayed if the countNumber is actually 10?

Reply:
Pae correct me if I'm wrong, but the fallback should be 0 as in file user.js

Reply:
Ah got it. had to look up the parseInt function!

Reply:
Same :sweat_smile: